app/controllers/v1/tasks_controller.rb

```diff
@@ -0,0 +1,91 @@
+module V1
+  class TasksController < ApplicationController
+    swagger_controller :tasks, 'Tasks'
+
+    swagger_api :index do
+      summary 'List all tasks'
+      notes 'This lists all the tasks belonging to the project_id'
+      param :query, :project_id, :uuid, :optional, 'tasks by project id , default all'
```

Comment:
Based on your route definitions it looks like you expect `project_id` to be a `path` param, not a `query` param.